app/MMB/src/scenes/tripServices/TripServiceRefreshContainer.js

```diff
@@ -78,12 +78,7 @@ class TripServiceRefreshContainer extends React.Component<Props, State> {
       {/* TODO: ordered services - how does it work here? */}
 
       <TitledMenuGroup
-        title={
-          <Translation
-            id="mmb.trip_services.general_services"
-            textTransform="uppercase"
```

Comment:
Why are you removing this?

Reply:
Aha, I see

Reply:
I am just removing the `textTransform="uppercase"` as it is now applied to all similar components